lms/djangoapps/courseware/views.py

```diff
@@ -423,6 +437,11 @@ def _index_bulk_op(request, course_key, chapter, section, position):
 
         context['show_chat'] = show_chat
 
+        if settings.FEATURES.get('ENTRANCE_EXAMS', False) and getattr(course, 'entrance_exam_enabled', False):
+            _, exam_current_score = get_entrance_exam_content_info(request, course_module, course)
+            if exam_current_score is not None:
+                context['entrance_exam_current_score'] = exam_current_score
+
```

Comment:
OK, spoke to @mattdrayer about get_course_milestones_fulfillment_paths() and when the user has completed the exam, this will return an empty set, so we're all set here